wherescape/connectors/hubspot/python_hubspot_check_missing_columns

```diff
@@ -0,0 +1,130 @@
+
+
+def create_table_rows(
+    missing_in_wherescape, missing_in_hubspot, environment, table_name
+):
+    # Creating the table row list for the insert query
+    table_rows = []
+    if len(missing_in_wherescape) > 0:
+        for row in missing_in_wherescape:
+            table_rows.append((environment, row, table_name, "wherescape"))
+    if len(missing_in_hubspot) > 0:
+        for row in missing_in_hubspot:
+            table_rows.append((environment, row, table_name, "hubspot"))
+
+    return table_rows
+
+
+def get_ds_rows(wherescape_instance):
+    # Get the datastore rows for comparison with the new load rows
+    table_name = f"datastore.ds{wherescape_instance.table[4:]}"
+    sql = f"select hubspot_environment, column_name, hubspot_load_table_name, missing_where from {table_name}"  # noqa: E501
+    rows = wherescape_instance.query_target(sql)
+    return rows
+
+
+def hubspot_check_missing_columns():
+    # Main function to check missing Hubspot columns
+    wherescape_instance = WhereScape()
+    hubspot_access_token_voys = wherescape_instance.read_parameter(
+        "hubspot_access_token_voys"
+    )
+    hubspot_instance = Hubspot(hubspot_access_token_voys)
+    environments = ("voys", "vg")
+    table_rows = []
+
+    # Query the hubspot api per environment and object_type and compare with
+    # the wherescape objects.
+    for environment in environments:
+        for object_type in OBJECT_TYPES_TO_CHECK:
+            table_name = "load_hubspot_%s_%s" % (environment, object_type)
+            sql = "SELECT lt_obj_key FROM ws_load_tab WHERE lt_table_name = ?"
+            results = wherescape_instance.query_meta(sql, [table_name])
+
+            if len(results) > 0:
+                lc_obj_key = results[0][0]
+                sql = "SELECT lc_col_name FROM ws_load_col WHERE lc_obj_key = ?"
+                results = wherescape_instance.query_meta(sql, [lc_obj_key])
+                wherescape_columns = [item[0] for item in results]
+                api_response = None
+                try:
+                    api_response = (
+                        hubspot_instance.client.crm.properties.core_api.get_all(
+                            object_type=object_type, archived=False
+                        )
+                    )
+                except ForbiddenException:
+                    logging.info(
+                        f"No access for type {object_type} via the Hubspot api for {environment}"  # noqa: E501
+                    )
+                else:
+                    api_results = api_response.to_dict()
+                    hubspot_columns = []
+                    for result in api_results["results"]:
+                        hubspot_columns.append(result["name"])
+                    missing_in_wherescape, missing_in_hubspot = compare_columns(
+                        wherescape_columns, hubspot_columns
+                    )
+                    table_rows = create_table_rows(
```

Comment:
these get overwritten in each loop. Shouldn't they get appended to some overall result?